src/libs/ReportActionsUtils.js

```diff
@@ -76,18 +76,20 @@ function getSortedReportActions(reportActions, shouldSortInDescendingOrder = fal
 }
 
 /**
- * Finds most recent IOU report action number.
+ * Finds most recent IOU request action ID.
  *
  * @param {Array} reportActions
  * @returns {String}
  */
-function getMostRecentIOUReportActionID(reportActions) {
-    const iouActions = _.where(reportActions, {actionName: CONST.REPORT.ACTIONS.TYPE.IOU});
-    if (_.isEmpty(iouActions)) {
+function getMostRecentIOURequestActionID(reportActions) {
+    const iouRequestActions = _.filter(reportActions, action => action.originalMessage
+        && action.originalMessage.type && action.originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.CREATE);
```

Comment:
This is the safe and easy way to access those subproperties. You may need to import `lodashGet` if it's not imported already.

```suggestion
    const iouRequestActions = _.filter(reportActions, action => lodashGet(action, 'originalMessage.type') === CONST.IOU.REPORT_ACTION_TYPE.CREATE);
```

Reply:
Ahhh thanks, I knew there was a better way. LODASH escaped me yesterday